gap/Internal.g

```diff
@@ -40,6 +62,21 @@ SimplifyWord@ := function(G, w, lmax)
         od;
     od;
 
+    # --------------------------------
+    # Check for option simplifyMethod:
+    # -------------------------------- 
+    # (option available in Internal_kbmagExtension.g)
+
+    # Construct ID:
+    # -------------
+    # for memoize function in SimplifyMethodMessenger@, (string to positive integer)
+    ID := AbsInt(CrcString(JoinStringsWithSeparator([String(gens), String(RelatorsOfFpGroup(G))],"")));
```

Comment:
Why do you not include the option `simplyMethod` (from the line below) in the ID as well to distinguish calls with different method set?